jenkins/bootstrap_test.py

```diff
@@ -1447,7 +1463,14 @@ def testValidTimeout(self):
                 if 'KUBEKINS_TIMEOUT=' not in line:
                     continue
                 found_timeout = True
-                mat = re.match(r'export KUBEKINS_TIMEOUT="(\d+)m".*', line)
+                if job.endswith('.sh'):
+                    mat = re.match(r'export KUBEKINS_TIMEOUT="(\d+)m".*', line)
+                else:
+                    mat = re.match(r'KUBEKINS_TIMEOUT="(\d+)m".*', line)
+                    realjob = self.GetRealBootstrapJob(job)
```

Comment:
This doesn't seem like a super great strategy either. Specifically you're now depending on the fact that testValidTimeout runs after testBootstrapCIYaml based on where they are defined in this file, which does not seem like a great practice.